doc/user/content/guides/cdc-postgres.md

```diff
@@ -46,12 +48,12 @@ Before creating a source in Materialize, you need to ensure that the upstream da
 1. Set the replica identity value for the tables you want to replicate:
 
     ```sql
-    ALTER TABLE repl_table REPLICA IDENTITY FULL;
+    ALTER TABLE repl_table REPLICA IDENTITY DEFAULT;
```

Comment:
I'm not entirely sure that this is correct for the direct Postgres source. To use `DEFAULT` the source has to work in upsert mode and the source tables would have to have primary keys defined, so unless this was tested I'd like to double check the behaviour.